cpp/tests/io/parquet_test.cpp

```diff
@@ -4279,6 +4279,9 @@ TEST_F(ParquetWriterTest, CheckColumnOffsetIndexNulls)
       auto const ci    = read_column_index(source, chunk);
       auto const stats = parse_statistics(chunk);
 
+      // should be half nulls, except no nulls in column 0
+      EXPECT_EQ(stats.null_count, c > 0 ? num_rows / 2 : 0);
```

Comment:
I weakly prefer this spelling:
```suggestion
      EXPECT_EQ(stats.null_count, c == 0 ? 0 : num_rows / 2);
```